src/webgpu/api/validation/createPipelineLayout.spec.ts

```diff
@@ -85,8 +85,12 @@ g.test('visibility_and_dynamic_offsets')
     const { type, hasDynamicOffset, visibility } = t.params;
     const info = kBindingTypeInfo[type as GPUBindingType];
 
+    const storageTextureFormat =
+      type === 'readonly-storage-texture' || type === 'writeonly-storage-texture'
```

Comment:
nit: can be `info.resource === 'storageBuf'`

Reply:
Done